checks.d/elastic.py

```diff
@@ -163,6 +163,8 @@ class ESCheck(AgentCheck):
         "elasticsearch.http.total_opened": ("gauge", "http.total_opened"),
         "jvm.mem.heap_committed": ("gauge", "jvm.mem.heap_committed_in_bytes"),
         "jvm.mem.heap_used": ("gauge", "jvm.mem.heap_used_in_bytes"),
+        "jvm.mem.heap_used_in_percents": ("gauge", "jvm.mem.heap_used_percent"),
```

Comment:
Actually I'd love to remove it or find a shorter name but `jvm.mem.heap_used` is already refering to another metric :/

Should I find a nicer name maybe ? Like `heap_used_ratio` or `heap_used_percentage` ?